gnodeb/worker/gnbcpueworker/handler.go

```diff
@@ -42,10 +42,20 @@ func HandleInitialUEMessage(gnbue *gnbctx.GnbCpUe,
 	intfcMsg common.InterfaceMessage) {
 
 	msg := intfcMsg.(*common.UuMessage)
-	sendMsg, err := test.GetInitialUEMessage(gnbue.GnbUeNgapId, msg.NasPdus[0], "")
-	if err != nil {
-		gnbue.Log.Errorln("GetInitialUEMessage failed:", err)
-		return
+	var sendMsg []byte
```

Comment:
Function name is initialUEMessage. So we should not do any change here. Logically caller should decide if its initial message or UL message.

Reply:
agree. reverted the change as it is not required for this procedure